docs/projects/template-fps/tutorial.md

```diff
@@ -197,7 +197,7 @@ You can regenerate the `default.snapshot` file from the SpatialOS menu option in
 
 While they are human-readable you are able to manually edit the values of the properties within, however be careful not to make mistakes that will inhibit the conversion back to binary form!<%(/Expandable)%>
 
-If you launch a local deployment (`Ctrl + L` in Unity), you should be able to see one `HealthPickup` entity in the world view of the [Inspector](http://localhost:21000/inspector/). You won't see the pickup in-game yet - this is the next step.
+If you launch a local deployment (`Ctrl + L` in Unity), you should be able to see one `HealthPickup` entity in the world view of the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/inspector). You won't see the pickup in-game yet - this is the next step.
```

Comment:
Is there a reason we linking to the Inspector itself?  
A tutorial usually links to the actual thing rather than docs about the thing.

Reply:
+1

linking to the thing lets people just follow along and click a button on the page to see their local deployment

Reply:
because the localhost might not be ready at that point. if a local deployment is not up, that link won't work. and we link to localhost later again when we actually instructed them to start a local deployment

Reply:
oh right yeah - that's a good point